lib/vicadmin/validate.go

```diff
@@ -62,8 +62,9 @@ type Validator struct {
 }
 
 const (
-	GoodStatus = template.HTML(`<i class="icon-ok"></i>`)
-	BadStatus  = template.HTML(`<i class="icon-attention"></i>`)
+	GoodStatus     = template.HTML(`<i class="icon-ok"></i>`)
+	BadStatus      = template.HTML(`<i class="icon-attention"></i>`)
+	DefaultVCHName = `VCH`
```

Comment:
Can we use something that makes it clear that it's a failure to resolve the name, rather than something that could be an actual name - otherwise we risk confusion if there's a conflict.